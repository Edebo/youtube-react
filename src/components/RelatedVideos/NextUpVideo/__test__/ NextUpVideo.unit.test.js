import React from "react";
import { shallow } from "enzyme";
import { NextUpVideo } from "../NextUpVideo";

describe("testing nextvideo component", () => {
  test("rendering the next up component", () => {
    const wrapper = shallow(<NextUpVideo />);
    expect(wrapper).toMatchSnapshot();
  });
});
