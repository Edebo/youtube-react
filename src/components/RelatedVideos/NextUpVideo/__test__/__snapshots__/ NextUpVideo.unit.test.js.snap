// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing nextvideo component rendering the next up component 1`] = `
<Fragment>
  <div
    className="next-up-container"
  >
    <h4>
      Up next
    </h4>
    <div
      className="up-next-toggle"
    >
      <span>
        Autoplay
      </span>
      <Checkbox
        defaultChecked={true}
        toggle={true}
        type="checkbox"
      />
    </div>
  </div>
  <VideoPreview
    horizontal={true}
  />
  <Divider />
</Fragment>
`;
